generic_server.cc

```diff
@@ -366,6 +367,7 @@ server::listen(socket_address addr, std::shared_ptr<seastar::tls::credentials_bu
 
 future<> server::do_accepts(int which, bool keepalive, socket_address server_addr) {
     while (!_gate.is_closed()) {
+        co_await coroutine::switch_to(get_scheduling_group());
```

Comment:
I am not sure what this is supposed to do, it calls seastar's current_sheduling_group, when changing to current has any effect?